quickwit-cli/tests/cli.rs

```diff
@@ -507,3 +507,39 @@ async fn test_all_with_s3_localstack_internal_api() -> Result<()> {
 
     Ok(())
 }
+
+// TODO add failure test
+#[tokio::test]
+async fn test_failure_not() -> Result<()> {
```

Comment:
what is this testing?

Reply:
nothing yet I was waiting for minoru's pr